app.js

```diff
@@ -11,63 +11,72 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let head = 0;
+let middle = 0;
+let bottom = 0;
 // set state for all of the character's catchphrases
+let catchphrases = [];
 
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
+    //console.log('clicking head', headDropdown.value);
     // increment the head change count state
-    
+    head++;
     // update the dom for the head (use style.backgroundImage on the headEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    headEl.style.backgroundImage = `url('./assets/${headDropdown.value}-head.png')`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
-
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    //console.log('clicking middle', middleDropdown.value);
     // increment the middle change count state
-    
+    middle++;
     // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    middleEl.style.backgroundImage = `url('./assets/${middleDropdown.value}-middle.png')`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
-
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    //console.log('clicking bottom', bottomDropdown.value);
```

Comment:
Awesome job using those console logs to check things are working.  Once it's all good you can get rid of these though.